timemachine/fe/rbfe.py

```diff
@@ -562,6 +569,24 @@ def estimate_relative_free_energy(
 
 
 def run_pair(mol_a, mol_b, core, forcefield, protein, n_frames, seed, n_eq_steps=10000):
+
+    # vacuum leg
+    vacuum_host_config = None
+    vacuum_res = estimate_relative_free_energy(
+        mol_a,
+        mol_b,
+        core,
+        forcefield,
+        vacuum_host_config,
+        seed,
+        n_frames=n_frames,
+        prefix="vacuum",
+        n_eq_steps=n_eq_steps,
+    )
+
+    # vacuum error should be below a threshold, otherwise this edge is likely problematic
+    assert np.linalg.norm(vacuum_res.all_dGs) < 1.0
+
```

Comment:
I'd rather have this as a separate function that just returns the vacuum_res. Then caller can decide what checks should be done and how to handle them.

Reply:
Or just call estimate_relative_free_energy directly.